src/distrs/binom.jl

```diff
@@ -11,3 +11,21 @@ import .RFunctions:
     binominvccdf,
     binominvlogcdf,
     binominvlogccdf
+
+# pdf for numbers with generic types
+binompdf(n::Real, p::Real, k::Number) = begin
+    # NOTE: fail to do below
+    #       prod(map(i -> (n + 1 - i) / i, 1:Int(k.))) * p^k * (1 - p)^(n - k)
+    n_choose_k = 1; i = 1;
+    while i <= k n_choose_k *= (n + 1 - i) / i; i += 1 end
+    n_choose_k * p^k * (1 - p)^(n - k)
+end
+
+# logpdf for numbers with generic types
+binomlogpdf(n::Real, p::Real, k::Number) = begin
```

Comment:
Do you want me to change all of them (there are other in those call-backs I implemented before)?